gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1499,6 +1499,28 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
   return TRUE;
 }
 
+/**
+ * @brief Parse caps and set tensors config (for other/tensors)
+ * @param config tensors config structure to be filled
+ * @param caps incoming capability
+ * @return TRUE/FALSE (if successfully configured, return TRUE)
+ */
+gboolean
+gst_tensors_config_from_cap (GstTensorsConfig * config, GstCaps * caps)
```

Comment:
caps can be const ptr `const GstCaps * caps`

Reply:
Updated! Thank you :)